spec/models/card_spec.rb

```diff
@@ -2,23 +2,23 @@
 
 describe Card do
   it "Check translation of the card in the different register" do 
-    card = Card.create(original: "cherry", translated: "Вишня", review_date: "2015-04-10 00:00:00 UTC")
+    card = Card.create(original: "cherry", translated: "Вишня", review_date: "2015-04-10 00:00:00 UTC", user_id: 1)
     expect(card.check_translation("CheRrY")).to be true
   end
 
   it "Returns false for incorrect translation" do
-    card = Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-10 00:00:00 UTC")
+    card = Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-10 00:00:00 UTC", user_id: 1)
     expect(card.check_translation("Shit")).to be false
   end
 
   context "The translation is true" do
-    let(:card) {Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-28 00:00:00 UTC")}
+    let(:card) {Card.create(original: "Hoogie code", translated: "Говнокод", review_date: "2015-04-28 00:00:00 UTC", user_id: 1)}
```

Comment:
Line is too long. [129/80]<br>Space missing inside {.<br>Space missing inside }.